index.js

```diff
@@ -1,2 +1,3 @@
 export {version} from "./build/package";
 export {default as geoStream} from "./src/stream";
+export {default as geoLength} from "./src/length";
```

Comment:
Alphabetical order of file names, probably. It’s hard to sort the imported symbols since one file typically exports multiple symbols.